src/patternfly/components/Scrollspy/scrollspy.scss

```diff
@@ -0,0 +1,168 @@
+  --pf-c-scrollspy__link--before--BorderBottomWidth: 0;
+  --pf-c-scrollspy__link--before--BorderLeftWidth: 0;
+  --pf-c-scrollspy__link--before--BorderColor: transparent;
+  --pf-c-scrollspy__link--focus--before--BorderTopWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-scrollspy__link--focus--before--BorderLeftWidth: 0;
+  --pf-c-scrollspy__link--focus--before--BorderColor: var(--pf-global--primary-color--100);
+  --pf-c-scrollspy__item--m-current__link--before--BorderTopWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-scrollspy__item--m-current__link--before--BorderLeftWidth: 0;
+  --pf-c-scrollspy__item--m-current__link--before--BorderColor: var(--pf-global--primary-color--100);
+  --pf-c-scrollspy--m-vertical__link--focus--before--BorderTopWidth: 0;
+  --pf-c-scrollspy--m-vertical__link--focus--before--BorderLeftWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-scrollspy--m-vertical__item--m-current__link--before--BorderTopWidth: 0;
+  --pf-c-scrollspy--m-vertical__item--m-current__link--before--BorderLeftWidth: var(--pf-global--BorderWidth--lg);
+
+  // text
+  --pf-c-scrollspy__item-text--Color: var(--pf-global--Color--200);
+  --pf-c-scrollspy__link--hover__item-text--Color: var(--pf-global--Color--100);
+  --pf-c-scrollspy__link--focus__item-text--Color: var(--pf-global--Color--100);
+  --pf-c-scrollspy__item--m-current__item-text--Color: var(--pf-global--Color--100);
+
+  // label
+  --pf-c-scrollspy__label--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-scrollspy__label--PaddingBottom: var(--pf-global--spacer--md);
+
+  padding-right: var(--pf-c-scrollspy--PaddingRight);
+  padding-left: var(--pf-c-scrollspy--PaddingLeft);
+  background-color: var(--pf-c-scrollspy--BackgroundColor);
+
+  &.pf-m-background {
+    --pf-c-scrollspy--BackgroundColor: var(--pf-c-scrollspy--m-background--BackgroundColor);
+  }
+
+  &.pf-m-center {
+    text-align: center;
+  }
+
+  &.pf-m-vertical {
+    --pf-c-scrollspy__list--PaddingRight: 0;
+    --pf-c-scrollspy__list--PaddingLeft: 0;
+    --pf-c-scrollspy__list--before--BorderTopWidth: var(--pf-c-scrollspy--m-vertical__list--before--BorderTopWidth);
+    --pf-c-scrollspy__list--before--BorderLeftWidth: var(--pf-c-scrollspy--m-vertical__list--before--BorderLeftWidth);
+    --pf-c-scrollspy__link--focus--before--BorderTopWidth: var(--pf-c-scrollspy--m-vertical__link--focus--before--BorderTopWidth);
+    --pf-c-scrollspy__link--focus--before--BorderLeftWidth: var(--pf-c-scrollspy--m-vertical__link--focus--before--BorderLeftWidth);
+    --pf-c-scrollspy__item--m-current__link--before--BorderTopWidth: var(--pf-c-scrollspy--m-vertical__item--m-current__link--before--BorderTopWidth);
+    --pf-c-scrollspy__item--m-current__link--before--BorderLeftWidth: var(--pf-c-scrollspy--m-vertical__item--m-current__link--before--BorderLeftWidth);
+
+    .pf-c-scrollspy__list {
+      flex-direction: column;
+    }
+  }
+}
+
+.pf-c-scrollspy__list {
+  position: relative;
+  display: inline-flex;
```

Comment:
Right now the container is a block element, but this list is inline. Seems like we would either want both the container and list to be inline (and the component will only be as wide as it needs to be), or both to be block (and the container it's in can manage the width)

current container
<img width="852" alt="Screen Shot 2020-10-21 at 10 57 41 AM" src="https://user-images.githubusercontent.com/35148959/96746056-68d3eb00-138c-11eb-8f93-a82db5e1d290.png">

current list item
<img width="457" alt="Screen Shot 2020-10-21 at 10 57 35 AM" src="https://user-images.githubusercontent.com/35148959/96746065-6b364500-138c-11eb-8be7-80db09490746.png">

if both were block
<img width="863" alt="Screen Shot 2020-10-21 at 10 56 24 AM" src="https://user-images.githubusercontent.com/35148959/96745776-1bf01480-138c-11eb-82f0-4aa9b3d0d4c9.png">

Reply:
Also right now the items' click area is different depending on the content, which we don't do in vertical lists AFAIK.

<img width="404" alt="Screen Shot 2020-10-21 at 11 08 22 AM" src="https://user-images.githubusercontent.com/35148959/96747439-e6e4c180-138d-11eb-9f32-c5480190a197.png">
<img width="448" alt="Screen Shot 2020-10-21 at 11 08 13 AM" src="https://user-images.githubusercontent.com/35148959/96747441-e77d5800-138d-11eb-9cef-85b1479e910c.png">